Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -299,6 +301,8 @@ import Foundation
                     return "Paid Price"
                 case .originalDownloadDate:
                     return "Original Download Date"
+                case .historyLatestPurchaseDate:
+                    return "Latest Purchase date"
```

Comment:
I think we are missing this one in the backend right? In that case, let's make sure uppercase/lowercase is consistent `Latest Purchase Date`

Reply:
```suggestion
                    return "Latest Purchase Date"
```

Reply:
It's there. Check `common_history_latest_purchase_date`